package.json

```diff
@@ -60,7 +60,7 @@
 				"to_build": false
 			},
 			"couchbase": {
-				"version": "28.4.0",
+				"version": "28.4.2",
```

Comment:
I bumped a second time because i forgot to remove cmake-js from production dependencies with 28.4.1

Reply:
I raran the full process after bump in buildMetadata